RedditSharp/Things/Post.cs

```diff
@@ -438,7 +457,7 @@ public IEnumerable<Comment> EnumerateComments()
                     }
                     if (things.Current is More)
                     {
-                        More more = (More)things.Current;
+                        var more = (More)things.Current;
```

Comment:
It was originally things.Current is More more from my changes so I changed it to what I thought was the previous one, but it should be fairly obvious IMHO that the var more is more, we should draft some syntax guidelines I guess